.github/ISSUE_TEMPLATE/major_change.md

```diff
@@ -29,12 +29,11 @@ The main points of the [Major Change Process][MCP] are as follows:
     * Compiler team members can initiate a check-off via `@rfcbot fcp merge` on either the MCP or the PR.
 * [ ] Once an MCP is seconded, the Final Comment Period begins. If no objections are raised after 10 days, the MCP is considered **approved**.
 
-You can read [more about Major Change Proposals on forge][MCP].
-
-# Comments
-
-**This issue is not meant to be used for technical discussion. There is a Zulip stream for that. Use this issue to leave procedural comments, such as volunteering to review, indicating that you second the proposal (or third, etc), or raising a concern that you would like to be addressed.**
+You can read [more about Major Change Proposals on forge][forge].
```

Comment:
Clicking on this link, I would expect to see content about MCPs not the general forge landing page.

Reply:
That's fair. I was fixing this to link to where it used to link before a refactor that footnoted links. It may be that that *isn't* where it should link. I can change that, and drop the outdated footnote.